metrics/firebase/firestore/rules/firestore.rules

```diff
@@ -3,30 +3,31 @@ service cloud.firestore {
   match /databases/{database}/documents {
 
     match /project_groups/{projectGroupId} {
-      allow read, delete: if isAccessAuthorized(database);
+      allow read: if isAccessAuthorized(database) || isSignInProviderAnonymous();
+      allow delete: if isAccessAuthorized(database);
       allow create, update: if isAccessAuthorized(database) && isProjectGroupValid();
     }
 
     match /projects/{projectId} {
-      allow read: if isAccessAuthorized(database);
+      allow read: if isAccessAuthorized(database) || isSignInProviderAnonymous();
       allow create, update: if isAccessAuthorized(database) && isProjectValid();
       allow delete: if false;
     }
 
     match /build/{buildId} {
-      allow read: if isAccessAuthorized(database);
+      allow read: if isAccessAuthorized(database) || isSignInProviderAnonymous();
       allow create, update: if isAccessAuthorized(database) && isBuildValid(database);
       allow delete: if false;
     }
 
     match /build_days/{buildDayId} {
-      allow read: if isAccessAuthorized(database);
+      allow read: if isAccessAuthorized(database) || isSignInProviderAnonymous();
```

Comment:
Should we improve naming a bit to make sure that it's readable & understandable how those 2 checks are different and what are the use cases?